bom-internal/pom.xml

```diff
@@ -28,7 +28,7 @@ This BOM's (dependency management element) is imported into the top level dockst
 dockstore-cli pom.xml files
 
 The dependencyManagement sections are located in the top level parent POM, and the
-dependecyManagement section is inherited by the other POMs that have the top level
```

Comment:
kinda neat that it can handle camel case like this

Reply:
It's pretty nice.

It can also handle compound words, like `dockstorewebservicetest` (but this feature is off by default as it can cause issues).